specs/vep-1739-multiple-python-versions/diagrams/high_level_design_plantUML_source.txt

```diff
@@ -0,0 +1,27 @@
+@startuml
+!include <awslib/AWSSimplified>
+!include <awslib/General/User>
+!include <awslib/Containers/ElasticContainerRegistry>
+!include <cloudinsight/file>
+
+caption Figure 1: High-level Design
```

Comment:
Add a more detailed caption that explains what the boxes are and what the arrows are. 

I used `legend ... endlegend` for that. It's up to you though.

Reply:
Removed the caption altogether and instead added a paragraph describing the diagram.